index.bs

```diff
@@ -2234,26 +2235,34 @@ The [=remote end steps=] with |command parameters| are:
   1. Let |type| be the value of the <code>type</code> field of
      |command parameters|.
 
+  1. Let |reference context id| be the value of the <code>context</code> field
+     of |command parameters|, if present, or null otherwise.
+
+  1. If |reference context id| is not null, let |reference context| be the
+     result of [=trying=] to [=get a browsing context=] with
+     |reference context id|. Otherwise let |reference context| be null.
+
   <!-- This is based on step 5 of https://w3c.github.io/webdriver/#new-window,
        but without using the "current browsing context" concept. -->
   1. Create a new [=top-level browsing context=] by running the [=window open
      steps=] with <var ignore>url</var> set to "<code>about:blank</code>",
      <var ignore>target</var> set to the empty string, and
      <var ignore>features</var> set to "<code>noopener</code>". This must be
      done without invoking the [=/focusing steps=] for the created browsing
-     context. If |type| is "<code>tab</code>", and the implementation supports
-     multiple browsing contexts in the same OS window, the new browsing context
-     should share an OS window with any other [=top-level browsing context=]. If
-     |type| is "<code>window</code>", and the implementation supports multiple
-     browsing contexts in separate OS windows, the created browsing context
-     should be in a new OS window. In all other cases the details of how the
-     browsing context is presented to the user are implementation defined.
-
-     Issue: HTML's [=window open steps=] depend on the "entry global object"
-     and WebDriver's <a href="https://w3c.github.io/webdriver/#new-window">New
-     Window</a> command depends on the "current browsing context". Given that we
-     don't have any such implicit context here, what OS window should new tabs
-     be opened in?
+     context. Which OS window the new [=/browsing context=] is created in
+     depends on |type| and |reference context|:
+
+     * If |type| is "<code>tab</code>" and the implementation supports
+       multiple browsing contexts in the same OS window, the new browsing context
+       should share an OS window with |reference context| if not null, and any
+       other [=top-level browsing context=] otherwise.
+
+     * If |type| is "<code>window</code>", and the implementation supports
+       multiple browsing contexts in separate OS windows, the created browsing
+       context should be in a new OS window.
+
+     * Otherwise, the details of how the browsing context is presented to the
+       user are implementation defined.
```

Comment:
What's not clear to me here is if this could also be a possible error. Or should we assume that the reverse type might always work? Probably not, so I would propose to add a paragraph that describes what needs to happen if no new browsing context can be created.

Given that the above mentioned issue is not tightly related to the changes in this PR maybe we should cover that in a separate one?

Reply:
An implementation that can't create new browsing contexts at all would just not implement `browsingContext.create`, right? Or are you thinking of some situation where it can succeed in some cases but fail in others?

Reply:
I think you can at least imagine an implementation that can create N browsing contexts, but not N+1. But I don't think we handle that in the WebDriver classic case and I'm not sure it should be a slocker to landing improvements here.